src/visitor/mysql.rs

```diff
@@ -895,4 +934,25 @@ mod tests {
             sql
         );
     }
+
+    #[test]
+    fn test_subselect_temp_table_wrapper() {
```

Comment:
Should we also have a test for deletes? Just copy this but make it to do `DELETE` so we also test that code path.

Reply:
added